test/trento_web/controllers/v2/cluster_controller_test.exs

```diff
@@ -22,7 +22,16 @@ defmodule TrentoWeb.V2.ClusterControllerTest do
     end
 
     test "should be compliant with Hana Cluster details schema", %{conn: conn} do
-      insert(:cluster, details: build(:hana_cluster_details))
+      insert(:cluster,
+        details:
+          build(:hana_cluster_details,
+            nodes:
+              build_list(1, :hana_cluster_node, %{
```

Comment:
changing the factory would remove the need to apply this change

Reply:
yep done